libpod/container_internal.go

```diff
@@ -758,6 +754,9 @@ func (c *Container) mountStorage() (err error) {
 			return errors.Wrapf(err, "error mounting storage for container %s", c.ID())
 		}
 	}
+	if c.state.Mounted {
```

Comment:
Any reason this moved? It seems like you added a condition to the SHMDir mount that otherwise doesn't need to be there if this remains at the top?

Reply:
We need to make sure the storage mount/unmount happens every time, else the counter does not get incremented/decrimented.